scripts/type-generation/test/copy.js

```diff
@@ -0,0 +1,46 @@
+const rimraf = require('rimraf')
+const copyfiles = require('copyfiles')
+const path = require('path')
+
+// TypeScript project root for testing particular typings
+const outDirs = ['sync', 'webdriverio']
+
+const packages = {
+    'webdriver': 'packages/webdriver',
+    '@wdio/sync': 'packages/wdio-sync',
+    'webdriverio': 'packages/webdriverio',
+    '@wdio/allure-reporter': 'packages/wdio-allure-reporter',
+
+    '@types/mocha': 'packages/wdio-mocha-framework/node_modules/@types/mocha',
+    '@wdio/mocha-framework': 'packages/wdio-mocha-framework',
+
+    '@types/jasmine/ts31': 'packages/wdio-jasmine-framework/node_modules/@types/jasmine/ts31',
+    '@types/jasmine': 'packages/wdio-jasmine-framework/node_modules/@types/jasmine',
+    '@wdio/jasmine-framework': 'packages/wdio-jasmine-framework',
+
+    '@types/selenium-standalone': 'packages/wdio-selenium-standalone-service/node_modules/@types/selenium-standalone',
+    '@wdio/selenium-standalone-service': 'packages/wdio-selenium-standalone-service',
+}
+
+/**
+ * copy package.json and typings from package to type-generation/test/.../node_modules
+ */
+async function copy() {
+    outDirs.forEach(async outDir => {
```

Comment:
You have to use either `for ... of` or `await Promise.all(packages.map())` construction. I prefer first option. `forEach` method does not in a proper way with async ... await https://stackoverflow.com/questions/37576685/using-async-await-with-a-foreach-loop

Reply:
keeping this open until this is resolved.

Reply:
my bad, fixed